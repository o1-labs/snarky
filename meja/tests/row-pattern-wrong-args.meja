let f = fun (x : [> `A(int)]) => {
  switch (x) {
    | `B => true
    | `A(b) => b
    | _ => false
  };
};
