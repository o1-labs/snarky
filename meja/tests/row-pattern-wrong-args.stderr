File "tests/row-pattern-wrong-args.meja", line 4, characters 15-16:
Error: Incompatible types bool and int:
       Cannot unify bool and int
